mcp/src/test/java/io/modelcontextprotocol/client/StdioMcpAsyncClientTests.java

```diff
@@ -25,7 +25,11 @@ protected McpClientTransport createMcpTransport() {
 		ServerParameters stdioParams = ServerParameters.builder("npx")
 			.args("-y", "@modelcontextprotocol/server-everything", "dir")
 			.build();
-		return new StdioClientTransport(stdioParams);
+		ServerParameters windowsStdioParams = ServerParameters.builder("cmd.exe")
```

Comment:
Perhaps you an move the logic into a single if statement. E.g. something like:
```java
ServerParameters stdioParams = null;
if (System.getProperty("os.name").toLowerCase().contains("win")) {
stdioParams = ServerParameters.builder("cmd.exe")
 .args("/c", "npx.cmd", "-y", "@modelcontextprotocol/server-everything", "dir")
			.build();
} else {
 stdioParams = ServerParameters.builder("npx")
			.args("-y", "@modelcontextprotocol/server-everything", "dir")
			.build();
}

Reply:
> Perhaps you an move the logic into a single if statement. E.g. something like:
> 
> ```java
> ServerParameters stdioParams = null;
> if (System.getProperty("os.name").toLowerCase().contains("win")) {
> stdioParams = ServerParameters.builder("cmd.exe")
>  .args("/c", "npx.cmd", "-y", "@modelcontextprotocol/server-everything", "dir")
> 			.build();
> } else {
>  stdioParams = ServerParameters.builder("npx")
> 			.args("-y", "@modelcontextprotocol/server-everything", "dir")
> 			.build();
> }
> ```

@tzolov No problem. I have revised both files again as your suggestion, please check again.